WordPress/gutenberg
# Theme.json: Add default values for settings.spacing.spacingScale

Follow-up on #41527

## What?
This PR adds the default value for `settings.spacing.spacingScale` among the spacing size presets defined in thme.json.

## Why?
Like many other presets, adding default values will help users set their own values, with auto-completion and validation.

## How?
I have defined default values based on [the default theme.json values in WordPress 6.1](https://github.com/WordPress/gutenberg/blob/bd3eb9bc7d1b63a24231a6e323414a862e0e85e3/lib/compat/wordpress-6.1/theme.json#L334-L340).

## Testing Instructions
Create a JSON file that references the JSON schema for this branch, such as:

```json
{
	"$schema": "https://raw.githubusercontent.com/WordPress/gutenberg/enhancement/theme-json-spacing-scale-default/schemas/json/theme.json",
	"version": 2,
	"settings": {
		"spacing": {
		}
	}
}
```

Under the `settings.spacing.spacingScale` property, confirm that default values are entered as initial values.
For the enum type, verify that the default value is indicated by the code editor.

## Screenshots or screencast 

https://user-images.githubusercontent.com/54422211/188468498-78d5437c-a832-44cc-a958-8c007c1abcc9.mp4
